package.json

```
@@ -223,7 +224,8 @@
             ".cu": "cd $dir && nvcc $fileName -o $fileNameWithoutExt && $dir$fileNameWithoutExt",
             ".ring": "ring",
             ".sml": "cd $dir && sml $fileName",
-            ".mojo": "mojo run"
+            ".mojo": "mojo run",
+            ".erl": "cd $dir && escript $fileName"
```

Simplify with below:
```suggestion
            ".erl": "escript"
```